src/components/Table/Table.tsx

```diff
@@ -176,7 +176,7 @@ export const Cell = styled(StyledBaseTD)`
   word-break: break-word;
   hyphens: auto;
   width: unset;
-  padding: 0.5em;
+  padding: 0.5em 0;
```

Comment:
Why this change?

Reply:
No need for horizontal padding on cells (makes it not line up with header)

Reply:
Also, half the padding comes from the Cell itself (the space between the text and the CellContainer) and half the padding went into the CellContainer (space between two rows)